src/mixing.html

```diff
@@ -15,53 +15,78 @@ <h2 id="mixing">Annotating MathML</h2>
+ MathML with the <q>intent</q> or (intended meaning) of the term.
  </p>
 
  <section>
   <h3 id="mixing_intent">The <code class="attribute">intent</code> attribute</h3>
-  <p>As decribed in <a href="#presm"></a> presentation MathML elements
+  <p>As decribed in <a href="#presm"></a> presentation <span class="ednote">Content?</span>
+  MathML elements
   (including [[MathML-Core]] allow attributes <code
   class="attribute">intent</code> and <code
   class="attribute">arg</code> that as described here allow the
-  <q>intent</q> or semantic meaning of the term to be specified. This
+  <dfn><q>intent</q></dfn> of the term to be specified. This
   annotation is not intented to provide a full mathematical definition
-  of the term but may be used guide Accessibility systems to give audio
+  of the term but may be used guide accessibility systems to give audio
   or braille renderings, see <a href="#accessibility"></a>. It may also
   be used to guide translations to Content MathML, or computational
   systems.</p>
 
+  <p>The <code class="attribute">intent</code> attribute encodes a
+  simple functional syntax representing  the intended meaning. A
+  formal grammar is given below, but a typical example would be
+  <code>intent="power($base,$exponent)"</code>. The parts consist:
+  of</p>
+  <ul>
+  <li><em>references</em> prefixed with <code>$</code>. A numeric
+  reference such as <code>$3/$1</code> references the [=intent=] of
+  the first child of the 3rd child of the element. An argumnt
+  reference such as <code>$name</code> refereces a descendent element
+  that has an attriute <code>arg="name"</code>. unlike <code class="attribute">id</code>
+  attributes, <code class="attribute">arg</code> do not have to be
+  unique within a document, when searching for a matching element the
+  search should only consider descendns and not consider descendents
+  of any element that has an <code class="attribute">intent</code> or
+  <code class="attribute">arg</code> attribute.</li>
+  <li><em>literal numbers</em> such as <code>2.5</code> denote
+  themselves. <span class="ednote">0.25e1?</span></li>
+  <li><em>literal identifiers</em> Non numeric literals reference
+  terms in the intent dictionary, currently located at <a
+  href="https://docs.google.com/spreadsheets/d/1EsWou1K5nxBdLPvQapdoA9h-s8lg_qjn8fJH64g9izQ/edit#gid=1358098730">Google
+  Sheets</a>. Note the list of [=intent=] identifiers is
+  <em>Open</em>: level 1 is a curated list maintained by the Math Working
+  Group and should be implemented by any implemenation of MathML
+  intent, so that suitable audio renderings of the level 1 terms will
+  be generated. Terms in other levels may be contributed; the
+  dictionary may give speech hints, but a system may also fall back on
+  reading the identifier name as given.</li>
+  <li><em>the wildcard</em>, <code>*</code> denotes all the children
+  <span class="ednote">just mi and mn children?</span> of the element so
```

Comment:
certainly not just mi and mn, since also msub, msup, msubsup, mover, munder, munderover, mrow, msqrt, mfrac, mroot, mtable, mstyle, mpadded, ... can contain valid arguments even to simple numeric functions

Reply:
yes I know:(  As I say above I think the specification of what this picks up has to fit in to a sentence, otherwise it has to go, I really don't want to have a whole section on a matching algorithm just on how the wildcard works. It's intended to make things simpler.  Open to suggestions........

Reply:
Indeed, `"every child except (possibly embellished) operators"` is better.

Not having the wildcard at all would be even nicer, given that n-ary infix operations can simply be left "as-is", without *any* intent annotation. Still waiting for counter-examples in #253 on that point.